Velez_Felipe/Asignments/Sum_list.py

```diff
@@ -0,0 +1,14 @@
+a = [1,2,5,10,255,3]
+# sum_of_a = sum(a)
+# print sum_of_a
+
+# sum_of_a = 0
+# for elem in a:
+# 	sum_of_a = sum_of_a + elem
+	
+# print sum_of_a
+sum_of_a = 0
+
+for index in range(len(a)):
+	sum_of_a = sum_of_a + a[index]
```

Comment:
since you're not interested in the index, you can iterate directly over elements. `for ele in a: sum_of_a += a`